pkg/lib/timeformat.ts

```diff
@@ -0,0 +1,77 @@
+/* Wrappers around Intl.DateTimeFormat and date-fns which use Cockpit's current locale, and define a few standard formats.
+ * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat
+ *
+ * Time stamps are given in milliseconds since the epoch.
+ */
+import cockpit from "cockpit";
+import { parse, formatDistanceToNow } from 'date-fns';
+import * as locales from 'date-fns/locale';
+
+// this needs to be dynamic, as some pages don't initialize cockpit.language right away
+export const dateFormatLang = (): string => cockpit.language.replace('_', '-');
+
+const dateFormatLangDateFns = (): string => {
+    if (cockpit.language == "en") return "enUS";
+    else return cockpit.language.replace('_', '');
+};
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const dateFnsLocale = (): any => (locales as any)[dateFormatLangDateFns()];
+
+type DTFOptions = {
+    [key: string]: string;
+};
+
+type Time = Date | number;
+
+// general Intl.DateTimeFormat formatter object
+export const formatter = (options?: DTFOptions) => new Intl.DateTimeFormat(dateFormatLang(), options);
+
+// common formatters; try to use these as much as possible, for UI consistency
+// 07:41 AM
+export const time = (t: Time): string => formatter({ timeStyle: "short" }).format(t);
+// 7:41:26 AM
+export const timeSeconds = (t: Time): string => formatter({ timeStyle: "medium" }).format(t);
```

Comment:
This added line is not executed by any test.

Reply:
I demand coverage!!

Reply:
... or the :axe:

Reply:
```
pkg/metrics/metrics.jsx:        let tooltip = timeformat.timeSeconds(time) + "\n\n";
```
we don't test the tooltip, but the qunit test does cover this.